android/jni/Android.mk

```diff
@@ -5,7 +5,7 @@ $(info TARGET_ARCH_ABI is $(TARGET_ARCH_ABI))
 
 $(info LOCAL_PATH is $(LOCAL_PATH))
 
-PREBUILT_LIBS := ./libs/prebuilt/android-$(TARGET_ARCH_ABI)
+PREBUILT_LIBS := ./libs/eegeo/prebuilt/android-$(TARGET_ARCH_ABI)

```

Comment:
Please can you explain the rationale behind moving the existing libs in libs/eegeo? Could we consider making a subdirectory for the newly provided external libs (eg. libs/external or libs/cardboard) as a less invasive alternative?